src/pages/index.tsx

```diff
@@ -1,34 +1,307 @@
+                  <Image
+                    src={mediumSucculentSVG}
+                    alt='medium succulent plant representing allocation in 2020'
+                    height='91.93px'
+                    width='96.79px'
+                  />
+                </Flex>
+
+                <Stack>
+                  <PageSection as='h4' textAlign='left' mb={2}>
+                    2020
+                  </PageSection>
+
+                  <Box>
+                    <PageText fontSize='h1' fontWeight={200} mb={1}>
+                      $12.9{' '}
+                      <PageText as='span' fontSize='stats'>
+                        million
+                      </PageText>
+                    </PageText>
+                  </Box>
+
+                  <PageText fontSize='faq.question' fontWeight={400} mb={1}>
+                    99 projects
+                  </PageText>
+                </Stack>
+              </Flex>
+
+              <Flex justifyContent='center' alignItems='center'>
+                <Flex w='150px' justifyContent='center' ml={2}>
+                  <Image
+                    src={bigSucculentSVG}
+                    alt='big opened succulent plant representing allocation in 2021'
+                    height='122.29px'
+                    width='131px'
+                  />
+                </Flex>
+
+                <Stack>
+                  <PageSection as='h4' textAlign='left' mb={2}>
+                    2021
+                  </PageSection>
+
+                  <Box>
+                    <PageText fontSize='h1' fontWeight={200} mb={1}>
+                      $24.0{' '}
+                      <PageText as='span' fontSize='stats'>
+                        million
+                      </PageText>
+                    </PageText>
+                  </Box>
+
+                  <PageText fontSize='faq.question' fontWeight={400} mb={1}>
+                    125 projects
+                  </PageText>
+                </Stack>
+              </Flex>
+            </Flex>
           </Flex>
-        </main>
-      </Flex>
-    </Container>
+        </section>
+
+        <section id='what-we-support'>
+          <Stack
+            borderRadius='10px'
+            bgGradient='linear(to-br, brand.ready.bgGradient.start 10%, brand.ready.bgGradient.end 100%)'
+            h={{ xs: '100%', md: '458px' }}
+            w='100%'
+            justifyContent='center'
+          >
+            <Flex
+              alignItems='center'
+              direction={{ base: 'column', md: 'row' }}
+              px={6}
+              py={{ xs: 10, md: 0 }}
+            >
+              <Stack mb={4} mr={{ base: 0, md: 12 }}>
+                <PageSection textAlign={{ base: 'center', md: 'left' }} mb={6}>
+                  What we support
+                </PageSection>
+
+                <Box>
+                  <Image
+                    src={isMobile ? whatWeSupportTreeMobileSVG : whatWeSupportTreeSVG}
```

Comment:
In general, I see low resolution images that make it difficult to read.

![image](https://user-images.githubusercontent.com/468158/151546911-536e57e3-f031-49b7-ae1a-c900ac296499.png)

I guess this is something we can tweak in the next `Image` component.

Reply:
Is it possible that maybe Netlify is compressing the images a bit? I see them sharper on local, I've applied some adjustments though, incresing the default quality to `85` to see it helps. Let me know what you think.

Reply:
Hmmm yes, something we could investigate later for sure (we could add a tech debt ticket for this).

Regarding Netlify doing that...not sure but I don't think so.

Did you try setting the quality to 99 or 100 to see if that actually fix the issue? just to confirm that the "problem" doesn't come from the Image component